src/features/singleRight/components/ResourceCollectionBar/ResourceCollectionBar.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Button } from '@digdir/design-system-react';
+import { MinusCircleIcon } from '@navikt/aksel-icons';
+import { useTranslation } from 'react-i18next';
+
+import type { CollectionBarProps } from '@/components';
+import { ActionBar, CollectionBar } from '@/components';
+import { type ServiceResource } from '@/rtk/features/singleRights/singleRightsApi';
+
+export interface ResourceCollectionBarProps
+  extends Pick<CollectionBarProps, 'proceedToPath' | 'compact'> {
+  /** The resources that are to appear in the CollectionBar content */
+  resources: ServiceResource[];
+  /** The callback function to be called when the remove button is pressed on a resource. */
+  onRemove: (identifier: string) => void;
+}
+
+/**
+ * A React component that displays a collection of resources using the CollectionBar component.
```

Comment:
But can you write something about when to use this compared to when to use CollectionBar? Because that's not clear to me 😊 Thinking of something like: "Use this component instead of CollectionBar when ...."

Reply:
But this component does use the CollectionBar, it just also renders the provided resources as small actionBars inside the CollectionBar's content (which is what I tried to explain in the line beneath this one, but apparently I didn't manage to get that point across 😅). The only reason this component exists as a separate entity is because of reusability, but I'm not sure that belongs in the JSdoc.

How about I write that "This component renders the provided resources as small actionBar's and inserts them into the contents of a CollectionBar. These actionBars have a remove-button which triggers the onRemove callback method."

Is that better?

Reply:
Yes and Instead of writing "These actionBars have a remove-button which triggers the onRemove callback method." I would write something like "this component does a lot of state setting that you otherwise would had to do manually. Color-change, remove-button and rendering of ActionBars works automatically".

Reply:
I mean.. technically it only sets a couple of props in the Collection bar that you don't already have to set as props to the ResourceCollectionBar so I don't know if that's the part I would focus on. It's the resources being rendered as ActionBars that's the important bit I would say.

How about this?
"This component renders a CollectionBar and populates its content with resources, rendered as removable ActionBars.
Colors and text is handled automatically inside the component"

Reply:
That's good!